project_index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+require_once('functions.php');
+$db = new PDO('mysql:host=db; dbname=collection_project', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `charname`, `class`, `level`, `strength`, `dexterity`, `constitution`, `intelligence`, `wisdom`, `charisma` FROM `characters`;");
+// $query->bindParam(':email', $email);
+$query->execute();
+$characters = $query->fetchAll();
+
+echo displayCharacters($characters);
```

Comment:
As per my comment on line 24, you should move this line to be inside the div tag you've got in the body